db/migrate/20210908204646_add_loan_prefix_to_response_set_kinds.rb

```diff
@@ -0,0 +1,9 @@
+class AddLoanPrefixToResponseSetKinds < ActiveRecord::Migration[6.1]
+  def up
+    execute("UPDATE response_sets SET kind = 'loan_' || kind")
+  end
+
+  def down
+    execute("UPDATE response_sets SET kind = substring(kind from 6)")
```

Comment:
it's been a long time since i've seen something 1 indexed! interesting sql does that.

Reply:
:)